internal/cluster/cluster_test.go

```diff
@@ -2372,6 +2375,7 @@ var _ = Describe("GenerateAdditionalManifests", func() {
 		manifestsGenerator.EXPECT().AddTelemeterManifest(ctx, gomock.Any(), &c).Return(nil)
 		manifestsGenerator.EXPECT().AddDiskEncryptionManifest(ctx, gomock.Any(), &c).Return(nil)
 		mockOperatorMgr.EXPECT().GenerateManifests(gomock.Any(), gomock.Any()).Return(nil).Times(1)
+		mockProviderRegistry.EXPECT().GenerateProviderManifests(gomock.Any(), gomock.Any()).Return(nil).Times(1)
```

Comment:
please add bad flows as well

Reply:
this is already implemented in latest push